src/numbers/xdr_large_int.js

```diff
@@ -176,6 +183,28 @@ export class XdrLargeInt {
     );
   }
 
+  /**
+   * @returns {xdr.ScVal} the smallest ScVal interpretation of the stored value
+   */
+  toScVal() {
+    switch (this.type) {
+      case 'i64':
+        return this.toI64();
+      case 'i128':
+        return this.toI128();
+      case 'i256':
+        return this.toI256();
+      case 'u64':
+        return this.toU64();
+      case 'u128':
+        return this.toU128();
+      case 'u256':
+        return this.toU256();
+      default:
+        throw TypeError(`invalid type: ${this.type}`);
+    }
+  }
```

Comment:
What is the use-case for this? Generally a contract would expect a specific type of scval.

Reply:
right, but if you do `ScInt(val, { type: 'i256' })` then `.toScVal()` is more convenient than remembering the type you passed to do `.toI256()`